code/game/machinery/vending/new_marine_vendors.dm

```diff
@@ -1029,21 +1030,21 @@ var/list/available_specialist_sets = list("Scout Set", "Sniper Set", "Demolition
 	spawned_gear_list = list(
 						/obj/item/clothing/head/helmet/marine,
 						/obj/item/clothing/under/marine,
-						/obj/item/clothing/shoes/marine
+						/obj/item/clothing/shoes/marine,
```

Comment:
Adding a comma at the end creates an empty entry.
We don't enforce not having a comma at the end because people forget, but there's zero need to purposely change this into the wrong way.